### PR TITLE
docs: Add a doc for how to add sphinx to your project.

### PR DESCRIPTION
**Pull request by feanil**
Distill the current recommendations into a quick how-to.

### CODE REVIEW

**Comment by feanil** [comment]
@mariajgrimaldi happy holidays!  I'd love your feedback on https://docsopenedxorg--239.org.readthedocs.build/en/239/developers/how-tos/add-sphinx-docs-to-a-repo.html but obviously I understand if it's a bit delayed. :-)

**Comment by mariajgrimaldi** [comment]
When generating a library using the cookie-cutter, a docs folder is created. Should we remove that folder entirely and replace it with what's generated with:
```
sphinx-quickstart --no-batchfile --extensions sphinxcontrib.contentui --extensions sphinx_copybutton --extensions sphinx.ext.graphviz --extensions sphinxcontrib.mermaid --no-sep -a "Open edX Community" -l "en" --release latest
```

**Comment by feanil** [comment]
@mariajgrimaldi I think that will be the easiest migration path, then you can move all the relevant RST files into the new folder.  Let me know if you think a different way might work better.

**Comment by mariajgrimaldi** [comment]
@feanil: I followed the instructions in openedx-events & filters and didn't encounter any issues. Thanks for this documentation! 
Just have a few more questions:
- I'm missing further setup for readthedocs hosting / configurations for openedx repos or a request mechanism. Should the specifics be included elsewhere? 
- I remember this [PR](https://github.com/openedx/openedx-events/pull/49) from openedx-events which fixed an issue with readthedocs, should we add a note about it?
- Should we have a post-merge task of updating the edx-cookiecutter to follow this new standard?

File: source/developers/how-tos/add-sphinx-docs-to-a-repo.rst [hunk 1]
```diff
@@ -0,0 +1,167 @@
...
+      ``repository_branch`` settings in ``html_theme_options`` but you should
+      review the rest of the settings as well.
+
+
+   .. code::
+
+      html_theme_options = {
+
+          "repository_url": TODO: Add a URL Here,
+          "repository_branch": TODO: Add the correct branch here
```
Comment (mariajgrimaldi) [inline source/developers/how-tos/add-sphinx-docs-to-a-repo.rst:99]:
```suggestion
          "repository_branch": TODO: Add the correct branch here,
```

**Comment by feanil** [comment]
Thanks for the feedback @mariajgrimaldi, yes, I've got https://github.com/orgs/openedx/projects/32/views/2?filterQuery=coo&pane=issue&itemId=16862232 to follow up and update this along with updating any existing docs creation in cookiecutters.

As for the RTD fix for django, I think I'll update the doc to include those lines.

**Pull request merged**